m3-libs/libm3/src/arith/POSIX/MathPosix.m3

```diff
@@ -0,0 +1,37 @@
+MODULE MathPosix EXPORTS Math;
+
+(*
+  Glue to fix broken prototypes in CM3-d5.11.9
+
+  Author : Mika Nystrom <mika.nystroem@intel.com>
+  June, 2024
+
+  Copyright (c) Intel Corporation, 2024.  All rights reserved.
+
+  See COPYRIGHT-INTEL for terms.  
+
+  SPDX-License-Identifier: BSD-3-Clause
+
+ *)
+
+IMPORT MathPosixC;
+FROM Ctypes IMPORT int;
+
+PROCEDURE frexp (x: LONGREAL;  VAR exp: int): LONGREAL =
+  BEGIN
+    exp := MathPosixC.frexp_exp_glue(x);
```

Comment:
I think "glue" and "C" are redundant, can just call it "C", but it is ok.

Reply:
Oh I can imainge maybe why you did this.  I'll keep looking.

Reply:
yeah so the reason for the extra layer comes down to something you did, so yes.. your feedback is welcome.

I think you were trying to rename these routines because the prototypes weren't matching what you could do in C.  I am guessing, though.

So what I tried to do was to ensure that the interface between M3 and C was as clean as possible, using only well-known types that are call-by-value.

But I can't change the Math.i3 interface because tons of code might depend on that.  So the extra M3 layer is really to convert from the Math.i3 interface to a simpler C interface.